src/test/resources/features/EditGame.feature

```diff
@@ -0,0 +1,69 @@
+Feature: Edit Game
+  In order to allow edit game
+  As an admin
+  I want to edit a game settings
+
+
+  Scenario: Edit game as admin
+    Given I login as "admin" with password "password"
+    And There is a game with id 1
+    When I edit the game with id 1
+    Then The response code is 201
```

Comment:
201 is the response code when a resource is created so does not make any sense here. The response to a successful edit should be a 200